### PR TITLE
Notify contributor's pack Github user

### PR DESCRIPTION
**Pull request by IkaDemisto**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold

## Related Issues
fixes: https://github.com/demisto/etc/issues/25229

## Description
Added default contributor's pack reviewers to pack metadata.

### CODE REVIEW

**Comment by IkaDemisto** [comment]
> ### Your contributed pack has been modified. Please review the changes.
> * @singlethreaded
> * @yyyogev
> * @crestdatasystems
> * @nathanhunstad
> * @MattHulse
> * @noamm91
> * @soumenchakraborty-endace
> * @Dan-at-Extrahop
> * @KGausel
> * @joshhe
> * @andrew-expanse
> * @asaf96
> * @thomasp11
> * @maor-b
> * @kashyap-solanki
> * @asiadeepinstinct
> * @Ngobo
> * @adiozer
> * @elebow
> * @sszilvasi

You are receiving this notification due to recent changes in content repo.
We have added notification on created PRs for default Github user that is defined in contributor's `pack_metadata` in case contributed pack files were modified.
If there is any problem with it, please let us know so we will remove your default Github user from pack.

File: Packs/Aella_StarLight/pack_metadata.json [hunk 1]
```diff
@@ -12,5 +12,6 @@
   ],
   "tags": [],
   "useCases": [],
-  "keywords": []
+  "keywords": [],
+  "githubUser": []
```
Comment (yaakovi) [inline Packs/Aella_StarLight/pack_metadata.json:16]: xwang95054

File: Packs/AlphaSOC_Network_Behavior_Analytics/pack_metadata.json [hunk 2]
```diff
@@ -12,5 +12,6 @@
     ],
     "tags": [],
     "useCases": [],
-    "keywords": []
+    "keywords": [],
+    "githubUser": []
```
Comment (yaakovi) [inline Packs/AlphaSOC_Network_Behavior_Analytics/pack_metadata.json:16]: ioj

File: Packs/AlphaSOC_Wisdom/pack_metadata.json [hunk 3]
```diff
@@ -12,5 +12,6 @@
     ],
     "tags": [],
     "useCases": [],
-    "keywords": []
+    "keywords": [],
+    "githubUser": []
```
Comment (yaakovi) [inline Packs/AlphaSOC_Wisdom/pack_metadata.json:16]: ioj

File: Packs/AlphaSOC_Network_Behavior_Analytics/pack_metadata.json (hunk 2 above)
Comment (IkaDemisto) [reply Packs/AlphaSOC_Network_Behavior_Analytics/pack_metadata.json:16]: it is username?

Comment (yaakovi) [reply Packs/AlphaSOC_Network_Behavior_Analytics/pack_metadata.json:16]: yes

**Comment by maor-b** [comment]
Hi,is this pull request related for us?

Thanks
Maor
________________________________

**Comment by fvigo** [comment]
@maor-b yes it was a minor change in the pack metadata.

**Comment by IkaDemisto** [comment]
Github username of pack contributor was added to `pack_metadata`. We concentrated it one PR, so all packs contributors will be aware of this new option (notification).

File: Packs/MinervaLabsAntiEvasionPlatform/pack_metadata.json [hunk 4]
```diff
@@ -1,16 +1,17 @@
...
+    "url": "https://minerva-labs.com/",
+    "email": "",
+    "created": "2020-04-14T00:00:00Z",
+    "categories": [
+        "Endpoint"
+    ],
+    "tags": [],
+    "useCases": [],
+    "keywords": [],
+    "githubUser": []
```
Comment (yaakovi) [inline Packs/MinervaLabsAntiEvasionPlatform/pack_metadata.json:16]: danikdanik

File: Packs/PerceptionPoint/pack_metadata.json [hunk 5]
```diff
@@ -12,5 +12,6 @@
     ],
     "tags": [],
     "useCases": [],
-    "keywords": []
+    "keywords": [],
+    "githubUser": []
```
Comment (yaakovi) [inline Packs/PerceptionPoint/pack_metadata.json:16]: yotam-tayeb

File: Packs/SCADAfence_CNM/pack_metadata.json [hunk 6]
```diff
@@ -12,5 +12,6 @@
   ],
   "tags": [],
   "useCases": [],
-  "keywords": []
+  "keywords": [],
+  "githubUser": []
```
Comment (yaakovi) [inline Packs/SCADAfence_CNM/pack_metadata.json:16]: AlexzxelA

File: Packs/ThreatQ/pack_metadata.json [hunk 7]
```diff
@@ -12,5 +12,6 @@
     ],
     "tags": [],
     "useCases": [],
-    "keywords": []
+    "keywords": [],
+    "githubUser": []
```
Comment (yaakovi) [inline Packs/ThreatQ/pack_metadata.json:16]: YoshaNir

File: Packs/PhishAI/pack_metadata.json [hunk 8]
```diff
@@ -12,5 +12,6 @@
   ],
   "tags": [],
   "useCases": [],
-  "keywords": []
+  "keywords": [],
+  "githubUser": []
```
Comment (yaakovi) [inline Packs/PhishAI/pack_metadata.json:16]: yevgenypats

File: Packs/CounterCraft/pack_metadata.json [hunk 9]
```diff
@@ -1,16 +1,17 @@
...
+    "url": "",
+    "email": "support@countercraft.zendesk.com",
+    "created": "2020-04-14T00:00:00Z",
+    "categories": [
+        "Deception"
+    ],
+    "tags": [],
+    "useCases": [],
+    "keywords": [],
+    "githubUser": []
```
Comment (yaakovi) [inline Packs/CounterCraft/pack_metadata.json:16]: tomac

**Comment by IkaDemisto** [comment]
@yaakovi are you merging this?

**Pull request merged**